RecoParticleFlow/PFSimProducer/plugins/PFSimParticleProducer.cc

```diff
@@ -146,16 +146,16 @@ void PFSimParticleProducer::produce(Event& iEvent,
     //getting the PCAloHit
     edm::Handle<edm::PCaloHitContainer> pcalohits;
     //   bool found_phit 
```

Comment:
Perhaps it is time to get rid of these commented out lines with outdated getByLabel calls